packages/@yoda/bluetooth/a2dp.js

```diff
@@ -208,12 +208,13 @@ BluetoothA2dp.prototype._onSourceEvent = function (data) {
 
 /**
  * @private
+ * @returns `true` if send success else `false`.
  */
 BluetoothA2dp.prototype._send = function (mode, cmdstr, props) {
   var data = Object.assign({ command: cmdstr }, props)
   var msg = [ JSON.stringify(data) ]
   var name = (mode === protocol.A2DP_MODE.SINK ? 'bluetooth.a2dpsink.command' : 'bluetooth.a2dpsource.command')
-  return this._flora.post(name, msg, floraFactory.MSGTYPE_INSTANT)
+  return this._flora.post(name, msg, floraFactory.MSGTYPE_INSTANT) === 0
```

Comment:
Could we simply return the raw status codes instead of booleans?

Reply:
I think RPC(flora) result code should not be dispatched to application layer.